addons/account/models/res_config_settings.py

```diff
@@ -220,6 +220,16 @@ class ResConfigSettings(models.TransientModel):
         compute='_compute_is_account_peppol_eligible',
     ) # technical field used for showing the Peppol settings conditionally
 
+    # Audit trail
+    check_account_audit_trail = fields.Boolean(string='Audit Trail', related='company_id.check_account_audit_trail', readonly=False)
+
+    @api.constrains('check_account_audit_trail')
```

Comment:
Note that it also depends on company_id

Reply:
Actually, I did this earlier (added company_id in constrains) but reverted it back. Because `check_account_audit_trail` is related to company_id (see in the field declaration)

Reply:
it works, but I think you're right that we should not do it in settings. I moved it to the company